sources/packages/api/src/route-controllers/institution/institution.institutions.controller.ts

```diff
@@ -84,17 +84,17 @@ export class InstitutionInstitutionsController extends BaseController {
   }
 
   /**
-   * Create institution on Institution setup process.
-   * @param payload
-   * @param userToken
-   * @returns Primary identifier of the created resource.
+   * Creates an institution during institution setup process when the
+   * institution profile and the user are created and associated altogether.
+   * @param payload information from the institution and the user.
+   * @returns primary identifier of the created resource.
```

Comment:
Please add param for userToken in the comments

Reply:
We had an agreement to avoid adding it 👍